test/unit/lib/plugins/aws/deploy/lib/checkForChanges.test.js

```diff
@@ -1503,22 +1234,47 @@ describe('test/unit/lib/plugins/aws/deploy/lib/checkForChanges.test.js', () => {
             headObject: async (params) => generateMatchingHeadObjectResponse(serverless, params),
           },
           CloudWatchLogs: {
-            describeSubscriptionFilters: () => {
-              // TODO: Ensure same ARN as lambda on which it is configured, but tweak index for one of the filters
-            },
-            // TODO: Configure stub
-            deleteSubscriptionFilter: null,
+            deleteSubscriptionFilter: deleteStub,
+            describeSubscriptionFilters: sandbox
+              .stub()
+              .onFirstCall()
+              .callsFake(() =>
+                Promise.resolve({
+                  subscriptionFilters: [
+                    {
+                      filterName: 'teststack-Fn1LogsSubscriptionFilterCloudWatchLog1-xxxxx',
+                      destinationArn: `arn:aws:lambda:us-east-1:999999999999:function:${
+                        serverless.service.getFunction('fn1').name
+                      }`,
+                    },
+                  ],
+                })
+              )
+              .onSecondCall()
```

Comment:
Why do we configure results for two calls here?

Reply:
We map the fix function on the logs group in the checkForChanges implementation, as we have two log groups there, we get two calls.